Cesium3DTilesSelection/include/Cesium3DTilesSelection/TileContentLoadResult.h

```diff
@@ -77,7 +77,7 @@ struct TileContentLoadResult {
   /**
    * @brief Contains the copyright information of the glTF model.
    */
-  std::optional<Cesium3DTilesSelection::Credit> credit{};
+  std::optional<std::vector<Cesium3DTilesSelection::Credit>> credits{};
```

Comment:
Is the `std::optional` necessary? I think an empty vector would be just as good.